packages/edit-post/src/components/sidebar/plugin-sidebar/index.js

```diff
@@ -130,44 +67,26 @@ function PluginSidebar( props ) {
  * 	</PluginSidebar>
  * );
  * ```
- *
- * @return {WPComponent} Plugin sidebar component.
  */
-export default compose(
-	withPluginContext( ( context, ownProps ) => {
+export default function PluginSidebarEditPost( { className, ...props } ) {
+	const { postTitle, shortcut } = useSelect( ( select ) => {
 		return {
-			icon: ownProps.icon || context.icon,
-			sidebarName: `${ context.name }/${ ownProps.name }`,
+			postTitle: select( 'core/editor' ).getEditedPostAttribute(
+				'title'
+			),
+			shortcut: select(
```

Comment:
Should all sites share the same shortcut for toggling the sidebar/complementary area?

Reply:
I think sharing the same shortcut makes sense. I guess we can keep the shortcut prop but in the future add a default value. In our core use cases we would not pass the prop so we would make all the screen share the same shortcut.